libs/ngx-dsfr/checkbox/src/lib/checkbox.module.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Angular imports
+ */
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+
+/**
+ * Internal imports
+ */
+import { DsfrCheckboxComponent } from './checkbox.component';
+import { TestHostComponent } from './testing/test-host.component';
+
+@NgModule({
+  imports: [CommonModule, FormsModule, ReactiveFormsModule],
```

Comment:
The way I see it, I don't think you need to import `FormsModule` and `ReactiveFormsModule`...
Could you try without them please ?